vital/types/metadata_tags.h

```diff
@@ -127,7 +127,7 @@
         bool,                                                           \
         "" )                                                            \
   CALL( SENSOR_LOCATION,                                                \
-        "Sensor Geodetic Location (lon/lat/meters)",                    \
+        "Sensor Geodetic Location (lon/lat/alt)",                       \
```

Comment:
This might be just a GitHub formatting thing, but it looks like this is adding some whitespace to the end of this line.  Can you remove that?

Reply:
I'm not sure what motivated the decision to add the trailing slashes, but each line is space-padded out to there. So the diff correctly reflects the additional required spaces to keep the trailing slashes aligned.

Reply:
I'm not super familiar with pre-processor wizardry, but I think these actually are needed to correctly do a multi-line macro.

Reply:
Hmmm , that is not really at the end of the line.  It just looks like trailing WS because the the old characters were replaced by spaces. The line terminates with a`\` which can not have any following spaces.

Reply:
My mistake, I overlook the trailing `\`.  I forgot this was a macro.